sale_blanket_order/wizard/create_sale_orders.py

```diff
@@ -95,6 +96,48 @@ def create_sale_order(self):
             'res_id': sale_order.id,
         }
 
+    @api.multi
+    def create_planned_sale_order(self):
+        self.ensure_one()
+        order_lines = []
+        if self.planned_date > self.blanket_order_id.validity_date:
+            raise UserError(_('planned date must be before the validity date of the blanker order: %s') %
+                            self.blanket_order_id.validity_date)
+
+        for line in self.lines_ids:
+            if line.qty == 0.0:
```

Comment:
please use float_compare method

Reply:
Done! Also changed it for the create_sale_order method in the wizard.